packages/styled-components/package.json

```diff
@@ -5,13 +5,27 @@
   "publishConfig": {
     "access": "public"
   },
-  "types": "dist/index.d.ts",
-  "main": "dist/styled-components.cjs.js",
-  "module": "./dist/styled-components.esm.js",
-  "react-native": "native/dist/styled-components.native.cjs.js",
-  "browser": {
-    "./dist/styled-components.cjs.js": "./dist/styled-components.browser.cjs.js",
-    "./dist/styled-components.esm.js": "./dist/styled-components.browser.esm.js"
+  "exports": {
```

Comment:
I think this can be simplied.

Here is the code snippet:

```
  "exports": {
    "types": "./dist/index.d.ts",
    "import": "./dist/styled-components.esm.js",
    "require": "./dist/styled-components.cjs.js",
    "react-native": "./native/dist/styled-components.native.cjs.js",
    "browser": {
      "import": "./dist/styled-components.browser.esm.js",
      "require": "./dist/styled-components.browser.cjs.js"
    }
  },
```

Reply:
Let me know what you think?